### DISCUSSION TITLE
Emulating Earth Observation from LEO

### DISCUSSION THREAD

**Discussion by ndwallace** [Stellarium/stellarium#3251] [Q&A]
Hi, I'm interested in potentially using Stellarium to simulate taking observations as if from a satellite in LEO around Earth. It seems to have a lot of good features in that regard, but I'm stuck on a few key issues.

Firstly, I've noticed that you can set the observer position to a desired lat,lon,alt relative to the Earth, however, the elevation caps at 200km, and the Earth itself is not visible when I do this (and only becomes so when I move the observer to another body - ie the Moon - and view it from there).

Is there a way to observe from a pose relative to the Earth with Earth model itself still visible? Also, can the elevation cap be altered - I did not see a setting for this in the config file, so I guess it's a hard-coded limit? (either that or I've just missed it)

Alternatively, I have a pre-computed orbital trajectory for a satellite; is there any way to import this into Stellarium somehow, and have the observer/view move along it? This could be as a series of lat,lon,alt positions relative to Earth, or as a TLE set (or equivalent). Perhaps adding a new Earth-orbiting body to Stellarium with the desired orbital parameters, to which I could then attach the observer to?

Any help or advice, even if its just to confirm whether this is or isn't something Stellarium can currently do, would be much appreciated.

Thanks in advance!

**Comment by 10110111** [comment]
I did an experiment a few months ago, adding the following to `ssystem_major.ini`:
```
[earth_orbiter]
albedo=0.
color=0., 0., 0.
halo=false
hidden=true
radius=0.001
name=Earth Orbiter
orbit_Inclination=0
orbit_MeanLongitude=0
orbit_Period=0.063
orbit_SemiMajorAxis=6800
parent=Earth
rot_obliquity=0
type=moon
```

This let me switch to "Planet" = "Earth Orbiter", but there's a problem: the Earth often disappears when some part of it gets out of the frame.

As for the elevation cap, this isn't going to work, because Stellarium renders a landscape instead of the current planet.

**Reply by gzotti** [reply]
I can estimate planet is clipped away when planet center is not on the screen. 

What @ndwallace wants has not been implemented. Yes, it would be nice to ride any satellite from the Satellites plugin. Somebody only has to write it...

**Reply by Atque** [reply]
> I can estimate planet is clipped away when planet center is not on the screen.
> 
> What @ndwallace wants has not been implemented. Yes, it would be nice to ride any satellite from the Satellites plugin. Somebody only has to write it...

We would need a much more hires texture of Earth for that.

**Reply by 10110111** [reply]
> We would need a much more hires texture of Earth for that.

We would need a proper model with atmosphere etc., but this is secondary. We could still have a look from LEO, even with the current poor model of the Earth.

**Comment by csaeiutezrwouvm** [comment]
best bridge!
